apps/backend/src/services/loginServices.ts

```diff
@@ -83,7 +83,8 @@ const generateToken = async (username: string, userId: number): Promise<string>
   const token = jsonwebtoken.sign(
     userDataForGeneratingToken,
     secret,
-    { expiresIn: '1h' }, // token expires in 1 hour
+    // { expiresIn: '1h' }, // token expires in 1 hour
+    { expiresIn: '1000h' }, // token expires in 1000 hours while dev
```

Comment:
I left the token valid for 1000 hrs because it became irritating login-in every once in a while. We should move it back to 1h later.